docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,69 @@
+---
+title: .NET API Analyzer
+description: Learn how the .NET API Analyzer can help detecting deprecated APIs and platform compatibility issues.
+author: oliag
+ms.author: mairaw
+ms.date: 12/11/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+---
+
+# .NET API Analyzer
+
+The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for APIs with different platforms and detects calls to deprecated APIs. It can be useful for any developers of the .NET family at any stage of development.
+
+API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb. The drop-down menu includes an option to suppress the warnings.
+
+## Discovering deprecated APIs
+
+### *What is deprecated API*
+*The .NET Framework is a large product that is getting constantly upgraded to better serve customer needs. It is natural to deprecate some APIs and replace them with new ones. Previously old APIs were marked with `[Obsolete]` attribute. There was only one diagnostic ID (CS0612) for all obsolete APIs that lead to one description for all cases and ability to suppress warnings for either all cases or none of them. To provide information and a way to suppress a warning per each API or a class of APIs, `Deprecation` term was introduced.*
+
+When a deprecated API, such as \<xref:System.Net.WebClient>, is used in a code, API Analyzer highlights it with a green squiggly line and shows a light bulb on the left as in the following example:
```

Comment:
I think the \ in front of the xref link is making the link broken.

Reply:
It is not marked Obsolete but it is considered deprecated. Here is discussion about it in the first question to this blog post: [https://blogs.msdn.microsoft.com/dotnet/2017/10/31/introducing-api-analyzer/](https://blogs.msdn.microsoft.com/dotnet/2017/10/31/introducing-api-analyzer/)

---
In reply to: [160491547](https://github.com/dotnet/docs/pull/3908#discussion_r160491547) [](ancestors = 160491547)